src/color.rs

```diff
@@ -27,8 +27,25 @@ pub enum Color {
 }
 
 fn truecolor_support() -> bool {
```

Comment:
Personally, I feel like putting conditional compilation *on* the function instead of *in* the function is a bit more readable.

```rust
#[cfg(target_os = "windows")]
fn truecolor_support() -> bool {
    // ...
}

#[cfg(not(target_os = "windows"))]
fn truecolor_support() -> bool {
    // ...
}
```